cli/lib/tasks/install.js

```diff
@@ -16,6 +17,89 @@ const unzip = require('./unzip')
 const logger = require('../logger')
 const { throwFormErrorText, errors } = require('../errors')
 
+const getNpmArgv = () => {
+  const json = process.env.npm_config_argv
+
+  if (!json) {
+    return
+  }
+
+  return JSON.parse(json).original || []
```

Comment:
Any chance `json` could be invalid and cause parsing to throw?

Reply:
Don't think so, but I added a try/catch to be safe.

Reply:
That was fast!